packages/tailwind/README.md

```diff
@@ -57,19 +57,37 @@ module.exports = {
 };
 ```
 
+## Fonts
+
+Fonts are automatically loaded from OBOS' CDN, so you don't have to host the font files yourself.
+
+If you use [Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP), you have to allow `https://www.obos.no` as a [font-src](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/font-src), otherwise the fonts will be blocked from loading.
```

Comment:
Dette må kanskje gjøres i helmet da og sette det som standard for både obos.no og obos.se?  Eller vil fontene ligge bak svensk cdn? 🤔

Reply:
Jepp, vi må legge det inn der. Tenker at vi bare laster fontene fra obos.no uanvhenig av hvilket domene du er på.